scheduler/scheduler-server/src/test/java/functionaltests/monitor/SchedulerMonitorsHandler.java

```diff
@@ -461,6 +475,29 @@ private void waitWithMonitor(EventMonitor monitor, long timeout) throws ProActiv
         throw new ProActiveTimeoutException("timeout elapsed");
     }
 
+    public void waitWithAnyMonitor(long timeout, EventMonitor... monitors) throws ProActiveTimeoutException {
+        TimeoutAccounter counter = TimeoutAccounter.getAccounter(timeout);
+        synchronized (monitors) {
+            Arrays.stream(monitors).forEach(monitor -> monitor.setTimeouted(false));
+
+            while (!counter.isTimeoutElapsed()) {
+                if (Arrays.stream(monitors).anyMatch(monitor -> monitor.eventOccured)) {
+                    return;
+                }
+
+                try {
+                    Thread.sleep(1000);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this use of "Thread.sleep()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2925)